ddtrace/_trace/product.py

```diff
@@ -145,6 +211,8 @@ def apm_tracing_rc(lib_config, dd_config):
             tags = dd_config._format_tags(lib_config["tracing_header_tags"])
         base_rc_config["_trace_http_header_tags"] = tags
 
+    if lib_config:
+        log.debug("Applied remote configs for APM Tracing: %s, Updated configs: %s", lib_config, base_rc_config)
```

Comment:
@rachelyangdog This debug log will make it easier to debug rc related issues. If this debug log doesn't exist then rc was not enabled.